flattentool/xml_output.py

```diff
@@ -13,7 +13,7 @@
     USING_LXML = False
     warn('Using stdlib etree may work, but is not supported. Please install lxml.')
 from flattentool.exceptions import DataErrorWarning
-from flattentool.sort_iati import sort_element, XMLSchemaWalker
```

Comment:
Thanks for this rename! I was thinking the same thing.

Reply:
oh, cool! I generally don’t do this sort of thing in external PRs, because renaming things is often a bit controversial!